### DISCUSSION TITLE
Go dependency error

### DISCUSSION THREAD

**Discussion by tripadvised** [evilsocket/opensnitch#551] [Development]
protocol folder only has a gitkeep file. Why is it imported as go module?
I am seeing this:
```could not import github.com/evilsocket/opensnitch/daemon/ui/protocol (no required module provides package "github.com/evilsocket/opensnitch/daemon/ui/protocol")compilerBrokenImport```

**Comment by gustavo-iniguez-goya** [comment]
You didn't compile the project properly: https://github.com/evilsocket/opensnitch/blob/wiki/wiki/Compilation.md

**Comment by tripadvised** [comment]
Compilation didn't happen because I'm using Rocky-linux. 

Libraries 

- pyqt5-dev-tools
- qttools5-dev-tools
- qt5-default 

has no yum/dnf counter parts. 

I have installed  

- libnetfilter_queue-devel <-- libnetfilter-queue-dev
- libpcap-devel <-- libpcap-dev

but for pyqt I tried to install pip packages and I see errors.



 ````   
Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/usr/lib64/python3.6/tokenize.py", line 452, in open
        buffer = _builtin_open(filename, 'rb')
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-build-e9_wx8p1/pyqt5/setup.py'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-e9_wx8p1/pyqt5/
 ````

**Reply by gustavo-iniguez-goya** [reply]
It looks like people install DEs with epel repositories [1]. Please, ask in the rocky linux forums about how to install pyqt libraries on that distro:

[1] https://forums.rockylinux.org/t/please-include-kde-in-rocky/529/16

**Reply by tripadvised** [reply]
turned out qt5-qtbase-devel is what I [needed](https://forums.rockylinux.org/t/installing-pyqt-libraries/4788/4?u=tripadvised).
